src/main/java/bio/terra/service/filedata/azure/tables/TableDirectoryDao.java

```diff
@@ -336,7 +340,9 @@ public void addEntriesToSnapshot(
       String datasetDirName,
       UUID snapshotId,
       List<String> fileIds) {
-    LRUMap<String, Boolean> pathMap = new LRUMap<>(MAX_FILTER_CLAUSES);
+    int cacheSize =
+        configurationService.getParameterValue(ConfigEnum.FIRESTORE_SNAPSHOT_CACHE_SIZE);
```

Comment:
Could we rename this to `SNAPSHOT_CACHE_SIZE`?

Reply:
Yes, can do. I don't think this is references in any of helm definitions, so should just be a simple rename.